### ISSUE TITLE
Launcher/Game should ask for RCT1 title music files if selected when missing

### ISSUE THREAD

**Issue by Patrik356b** [OpenRCT2/OpenRCT2#3965]
Like title says.

I´m surprised the launcher didn't have a button to browse for the rct1 install folder so the required files would be imported.

**Comment by Nubbie** [comment]
You want a prompt to pop up when you select the song so you can use it in-game?

**Comment by Patrik356b** [comment]
Something like that, yeah

**Comment by AaronVanGeffen** [comment]
To be honest, I think it'd be more confusing to ask for yet another data directory in the initial launch, especially considering it's optional.

At some point, we'll be moving to our own data format, though. I imagine more assets might be imported from RCT1 at that point, so it might be more viable then.

**Comment by Patrik356b** [comment]
no, not at initial launch, only if you change the title screen to RCT1 (parks or music) in the option when missing the required files
